iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/event/common/schema/PipeSchemaRegionWritePlanEvent.java

```diff
@@ -58,6 +60,26 @@ public PlanNode getPlanNode() {
     return planNode;
   }
 
+  public DeletionResource getDeletionResource() {
```

Comment:
If there are "deletionResource" in it, maybe use a new class to make it clearer?

Reply:
fixed, introduce PipeDeleteDataNodeEvent